packages/synapse-react-client/.storybook/main.ts

```diff
@@ -21,7 +21,7 @@ const config: StorybookConfig = {
     options: {
       strictMode: false,
       builder: {
-        viteConfigPath: './.storybook/storybook-vite.config.ts',
+        viteConfigPath: './.storybook/storybook-vite.config.mts',
```

Comment:
Moved Storybook Vite config to `.mts` so the Storybook manager treats it as an esm source